README.md

```diff
@@ -23,7 +23,7 @@ Viddy well, gopher. Viddy well.
     * diff highlight.
 * Time machine mode. 😎
     * Rewind like video.
-    * Go to the past, and back to the future.
+    * Go to the past, and back to the present.
```

Comment:
I like this description, can't it be interpreted as semantically correct? (I am not a native speaker, so I don't know, so these pointers are very helpful.

I believe time machine mode is the ability to go back to the past and return to any future point (of course, before present) seen from the past point.